tools/bin/publish_docker.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+# List of directories without "airbyte-" prefix.
+projectDir=( 
+  "workers"
+  "cli"
+  "webapp"
+  "server"
+  "temporal"
+  "container-orchestrator"
+  "config/init"
+  "bootloader"
+  "metrics/reporter"
+  "db/lib"
+  "scheduler/app"
+)
+
+# Set default values to required vars. If set in env, values will be taken from there.
+JDK_VERSION=${JDK_VERSION:-17.0.1}
```

Comment:
this syntax is always a pain for me.  Can you verify that you tested these locally?
when I try it, the default does not stick.  this output is from my command line
```
bash-3.2$ POSTGRES_IMAGE2=${POSTGRES_IMAGE2:-postgres:13-alpine} echo postgres2 is $POSTGRES_IMAGE2
postgres2 is
```

Reply:
Yes, I tested it locally. It works as expected. When you have env var, it overrides default value in script.

Reply:
I know the override works, it's the default that I'm worried about ☝️

Reply:
awesome, thanks @git-phu